packages/manager/apps/pci-gateway/src/api/hooks/data.ts

```diff
@@ -33,133 +34,118 @@ export type TSizeItem = {
   availableRegions: TAvailableRegion[];
 };
 
+export type TAvailablePlansGrouped = {
+  [key: string]: { code: string; regions: TPlanRegion[] };
+};
+
+const getPrice = (addon: TAddon): number | undefined =>
+  addon?.pricings.find((price) => price.capacities.includes('consumption'))
+    ?.price;
+
 export const useData = (projectId: string) => {
-  const { i18n, t: tSelector } = useTranslation('catalog-selector');
-  const { t: tRegion } = useTranslation('regions');
+  const { t } = useTranslation(['catalog-selector', 'regions']);
   const { data: cloudCatalog } = useCloudCatalog();
   const { data: availableGatewayPlans } = useAvailableGatewayPlans(projectId);
 
-  const { data: inactiveRegions } = useInactiveRegions(projectId);
+  const getBandWidthLabel = useCallback(
+    (bandwidth: number) =>
+      bandwidth > 1000
+        ? t(`pci_projects_project_gateways_model_selector_bandwidth`, {
+            bandwidth: bandwidth / 1000,
+          }) +
+          t(
+            'pci_projects_project_gateways_model_selector_bandwidth_unit_size_gbps',
+          )
+        : t(`pci_projects_project_gateways_model_selector_bandwidth`, {
+            bandwidth,
+          }) +
+          t(
+            'pci_projects_project_gateways_model_selector_bandwidth_unit_size_mbps',
+          ),
+    [],
```

Comment:
```suggestion
    [t],
```

Reply:
@Tsiorifamonjena 
Maybe you can use your IDE to be sure that your are not missing dependencies